src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/BundleEnvironmentGetCommandLineArgs.cs

```diff
@@ -29,6 +31,20 @@ public void GetEnvironmentArgs_0_Returns_Bundled_Executable_Path()
                 .Pass()
                 .And
                 .HaveStdOutContaining(singleFile);
+
```

Comment:
Hi @mateoatr  these tests should be separate facts. Thanks.

Reply:
Does it make sense to have the second test here? Feels a little bit weird to test the behavior of a non single-file app in this file.

Reply:
I agree that this is not a great factoring. I suggested having the related test here because we have the infrastructure (and we cannot yet test single-file easily from Libraries subset). But its only a suggestion.

However, if we do keep the two versions they should be separate facts, because unit tests should test only *one* thing. Thanks for making the change.
Thanks for making the change.